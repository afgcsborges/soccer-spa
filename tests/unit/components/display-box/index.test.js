import DisplayBox from 'components/display-box'
import React from 'react'
import { shallow } from 'enzyme'

it('should match snapshot,label and value', () => {
    const wrapper = shallow(<DisplayBox label="SOME_LABEL" value="SOME_VALUE" />)

    expect(wrapper).toMatchSnapshot()

    expect(wrapper.find('Text').at(0).props().label).toBe('SOME_LABEL')
    expect(wrapper.find('Text').at(1).props().label).toBe('SOME_VALUE')
})
