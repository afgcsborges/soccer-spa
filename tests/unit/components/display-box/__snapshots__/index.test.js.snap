// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot,label and value 1`] = `
<Space
  align="start"
  direction="horizontal"
  size="large"
>
  <styled.div>
    <Text
      label="SOME_LABEL"
      level={1}
      strong={false}
      underline={false}
    />
  </styled.div>
  <styled.div>
    <Text
      label="SOME_VALUE"
      level={1}
      strong={true}
      underline={false}
    />
  </styled.div>
</Space>
`;
